### PR TITLE
bug:[CORE-377]metadataforassetfilter

### PR DESCRIPTION
**Pull request by shefalibisht1992**
# Description

Adding metadat for asset fillters

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Chore (no code changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

**Review by dheerajkholia-paladin — APPROVED** [review]

File: installer/resources/pacbot_app/files/DB.sql [hunk 1]
```diff
@@ -2913,3 +2913,16 @@ update pac_v2_ui_options set optionType = "Long" where optionName="Violations" a
...
+INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL,optionType) VALUES (14,'Account Name','accountname.keyword','/compliance/v1/filters/attribute',"String");
+INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL,optionType) VALUES (14,'Region','region.keyword','/compliance/v1/filters/attribute',"String");
+INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL,optionType) VALUES (14,'Cloud Type','_cloudType.keyword','/compliance/v1/filters/attribute',"String");
+INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL,optionType) VALUES (14,'Environment','tags.Environment.keyword','/compliance/v1/filters/attribute',"String");
+INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL,optionType) VALUES (14,'Owner','tags.Owner.keyword','/compliance/v1/filters/attribute',"String");
+INSERT IGNORE INTO pac_v2_ui_options (filterId,optionName,optionValue,optionURL,optionType) VALUES (14,'Product','tags.Product.keyword','/compliance/v1/filters/attribute',"String");
```
Comment (dheerajkholia-paladin) [inline installer/resources/pacbot_app/files/DB.sql:2925]: These should be populated dynamically from mandatory tags config.

Comment (santhosh-challa) [reply installer/resources/pacbot_app/files/DB.sql:2925]: @shefalibisht1992 can you populate them?

Comment (shefalibisht1992) [reply installer/resources/pacbot_app/files/DB.sql:2925]: yes

**Comment by santhosh-challa** [comment]
@shefalibisht1992 I think we can close this right?

**Comment by shefalibisht1992** [comment]
Closing since raised other with PR review comments

**Pull request closed without merging**